gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -177,6 +177,7 @@
                 delete finalPayload.lifecycleInstructions;
                 $.extend(true, session, finalPayload);
             }
+            gpii.pcpInterface.sendUserMessage("New configuration driven by context changes");
```

Comment:
You have to use the IOC system to get a hold of the PCP, since when calling the method directly like you do here, there is no way for the PCP to get a hold of the socket. When fixing the RBMM->PCP messaging capabilities, I added some functions that might be useful - see: https://github.com/GPII/universal/pull/423/files (in particular the context manager file)